src/structures/MessageEmbed.js

```diff
@@ -37,6 +30,10 @@ class MessageEmbed {
    * @property {Partial<MessageEmbedFooter>} [footer] The footer of this embed
    */
 
+  /**
```

Comment:
Please revert documenting the second parameter of the constructor, it's intentionally undocumented because it's only to be used internally (since Discord doesn't hold itself to the limitation it's imposing).

Reply:
Addressed in https://github.com/discordjs/discord.js/pull/6629/commits/870d811245c0aa8a95ebd0543fbdca9412c0b735